book/src/remapping.md

```diff
@@ -48,7 +49,24 @@ Ctrl, Shift and Alt modifiers are encoded respectively with the prefixes
 | Null         | `"null"`       |
 | Escape       | `"esc"`        |
 
-Keys can be disabled by binding them to the `no_op` command.
+Individual keys can be disabled by binding them to the `no_op` command.
+
+To remove the default bindings for all modes defined by the user, `unbind-default-keys = true` can be added to the top level configuration.
+
+```toml
+unbind-default-keys = true
+
+# Only these normal mode bindings will be used
+[keys.normal]
+n = "normal_mode"
+t = "goto_definition"
+
+# No select mode bindings will be used
+[keys.select]
+
+# All default insert mode bindings will be used because new bindings weren't defined
```

Comment:
This behavior suprises me - I would expect `unbind-default-keys` to clear the keymap completely

Reply:
I changed it to no longer include this behaviour. The main reason for this was so users might forget to bind `esc = "normal_mode"` or similar, but I added a blurb about it in the docs instead